sdk/samples/iot/hub/linux/how_to_iot_hub_samples_linux.md

```diff
@@ -318,7 +316,7 @@ _The following was run on an Ubuntu Desktop 18.04 environment, but it also works
     On the Azure Portal,
     - Go to your Azure IoT hub page.
     - Click on "IoT devices" under "Explorers".
-    - From the list of devices, click on your device (created on step Z).
```

Comment:
There is no `step Z`, I am guessing this was meant to be `step 7`, but the step numbers were incorrect because there were duplicates, so I fixed those. With the fix, I believe this should be `step 6`. @ewertons, please review to make sure they refer to the right number now.

Reply:
step 6 looks correct